scripts/create_latest_tag.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
IMO all scripts for release should be inside the hack directory (maybe `hack/release`).
If not proper note should be added to [readme](https://github.com/kyma-project/helm-broker/blob/master/README.md#project-structure)

Reply:
moved to hack/release 👍